book/08_Flood_analysis/3_Retrieving_FloodData.md

```diff
@@ -89,6 +81,7 @@ opts = {
 
 
 ```python
+livingston_tx = Point(-95.09, 30.69)
 m = folium.Map(location=(livingston_tx.y, livingston_tx.x), control_scale = True, zoom_start=8)
```

Comment:
We can do this without Folium using GeoViews instead (which is built on top of Holoviews & hence more consistent with Hvplot in terms of API).

In the interest of minimising the number of tools we have to explain, let's change this visualization step (and likewise everywhere else).